docs/topics/mta-web-creating-applications.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-applications_{context}"]
+= Creating an application profile
+
+You can create an application profile in order to add an application to the *Application Invcentory* for assessment and analysis.
+
+[TIP]
```

Comment:
An idea: If this procedure is recommended to be done before creating the profile, then maybe place it in the doc BEFORE the profile creation procedure.

Reply:
That's a good idea -- it will have to wait until all the PRs for this guide are merged so i can create a cross-reference.

Reply:
@RichardHoch , ack, thanks.